### DISCUSSION TITLE
some functionality  transform issues

### DISCUSSION THREAD

**Discussion by ghost** [SVG-Edit/svgedit#536] [Q&A]
Hello programmer,I want to add this program to my own program and Add a little functionality to it like **rubber** **brush**,what should I do,thanks

**Comment by brettz9** [comment]
Hi,

You can either:

1. Add an extension that adds the feature you want
2. Directly modify the main editor HTML
3. Control the editor through the embedded API

Please take a look at the README and `docs` and as needed, the source code. If you have further support questions, please use Stackoverflow or such a Q&A site.
